lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
-  padding-top: 20px;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-xl $nx-spacing-l;
+  text-align: right;
 }
 
 .nx-form-group {
-  margin-bottom: 20px;
+  margin-bottom: 6px;
 }
 
 .nx-form-row {
   @include container-horizontal;
-  margin-bottom: 24px;
+  margin-bottom: 6px;
 
   .nx-form-group, .nx-fieldset {
     display: inline-block;
-    margin-right: 24px;
-    margin-bottom: 0;
-    vertical-align: bottom; // fixes issue where certain form types display lower
-
-    &:last-child {
-      margin-right: 0;
-    }
-  }
-
-  .nx-label {
+    margin-right: $nx-spacing-l;
     margin-bottom: 0;
   }
 }
 
-.nx-btn-bar--forms {
-  border-top: $nx-border-dark;
-  padding-top: 8px;
-}
-
 // labels
 
 .nx-label, .nx-legend {
   @include semi-bold();
 
   color: $nx-text-color;
-  display: block;
-  font-size: $nx-font-size;
-  margin-bottom: $nx-spacing-xs;
+  padding: 0;
 
   .disabled {
     color: $nx-grey-4;
```

Comment:
What does this do

Reply:
> What does this do

The browser agent adds 2px of padding to <legend>, I just removed it here for both for clarity.

Reply:
> > What does this do
> 
> The browser agent adds 2px of padding to , I just removed it here for both for clarity.

I think the question was about
```
  .disabled {
    color: $nx-grey-4;
  }
```

Reply:
> > > What does this do
> > 
> > 
> > The browser agent adds 2px of padding to , I just removed it here for both for clarity.
> 
> I think the question was about
> 
> ```
>   .disabled {
>     color: $nx-grey-4;
>   }
> ```

Ah, I was thrown off by the green highlight above it, yeah that value is not longer required, removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/019733681eb9b5b1f86776237262b0795ea47123